qcodes/instrument/parameter.py

```diff
@@ -994,7 +1085,7 @@ def __init__(self, name: str,
 
         if not hasattr(self, 'set') and set_cmd is not False:
             if set_cmd is None:
-                self.set_raw: Callable = partial(self._save_val, validate=False)
+                self.set_raw: Callable = lambda x: x
```

Comment:
This is ok since set wrapper already calls save val right?

Reply:
And we do have tests for that I assume

Reply:
indeed. `_save_val` (now `_update_cache`) is part of 'set' and 'get' thanks to `wrap_get` and `wrap_set` anyway. Many tests implicitly test this for sure, but perhaps also explicitly.

let me add a test that goes inside the `._latest` and asserts stuff there just to be sure.

Reply:
done